projects/coordinator/src/test/java/org/batfish/coordinator/WorkMgrServiceTest.java

```diff
@@ -84,4 +91,57 @@ public void getNonEmptyContainer() throws Exception {
     expectedTestrigs.add("testrig2");
     assertThat(container.getTestrigs(), equalTo(expectedTestrigs));
   }
+
```

Comment:
I would simplify these tests dramatically, or drop them. They're way too redundant (almost the same 50 lines of code in each file) and complex.